### PR TITLE
Indi tax

### PR DESCRIPTION
**Pull request by arcisad**
We created a individual tax calculator class plus a test unit for testing the class. the individual.py file is in the source folder and the test unit is in the tests folder.

### CODE REVIEW

**Comment by hsorby** [comment]
Nice start with the commit messages.

**Comment by hsorby** [comment]
Great there is some documentation to.

File: source/drhtaxes/individual.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Tax(object):
+    """
+    This class takes income and loss as the input and claculates the individual tax based on tier income and loss.
+    """
+    def calc_tax(self,income):
```
Comment (hsorby) [inline source/drhtaxes/individual.py]: This function could also do with some documentation please.

**Review by hsorby — CHANGES_REQUESTED** [review]

**Comment by arcisad** [comment]
We have updated the individual.py file. We added comments for the functions in the class.

**Review by nickerso — APPROVED** [review]
Looks like reviewer comments have been addressed.

**Comment by hsorby** [comment]
Great, looks very good.

**Review by hsorby — APPROVED** [review]

**Pull request merged**